applications/rubintv/values-usdfdev.yaml

```diff
@@ -0,0 +1,89 @@
+siteTag: "usdf-dev"
+
+frontend:
+  debug: true
+  env:
+  - name: http_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: https_proxy
+    value: "http://sdfproxy.sdf.slac.stanford.edu:3128"
+  - name: no_proxy
+    value: ".sdf.slac.stanford.edu,.slac.stanford.edu,localhost,127.0.0.1"
+  image:
+    tag: tickets-DM-40149
+    pullPolicy: Always
+
+workers:
+  replicas: 1
+  image:
+    repository: lsstts/rubintv-broadcaster
+    tag: c0030_usdf
+    pullPolicy: Always
```

Comment:
I'm confused about why on tucson-teststand we get this package from ts-dockerhub but here we get it from Docker Hub. Where is this package actually maintained?

Reply:
...note that it's the _usdf tag, not merely c0030, and...beats me.  It's somewhere in the T&S Jenkins I assume.  I don't know where they push which images.  This particular part was on fire when I got here.  This is probably also why there's a pull secret in the config.

Reply:
And that's not a safe assumption.  Maybe Merlin builds and pushes it by hand; I wouldn't be the least bit surprised.